package.json

```diff
@@ -25,7 +25,7 @@
   "devDependencies": {
     "@amplitude/eslint-plugin-amplitude": "^1.0.1",
     "@aws-sdk/client-s3": "^3.229.0",
-    "compression-webpack-plugin": "^11.1.0",
+    "compression-webpack-plugin": "^10.0.0",
```

Comment:
I'd vote for updating Node in the workflows to latest LTS instead. Node 16 is EOL - https://endoflife.date/nodejs. Since it's just the build environment it shouldn't affect the min node version of the package

Reply:
Thanks, Justin. 
That's fair. I will update the node version then.

Reply:
Closed this and created a new pr to upgrade the node version.